scripts/screeps_rl_training_runner.py

```
@@ -1221,6 +1271,229 @@ def build_policy_update(
+def first_nonzero_tier_gradient(tier_gradients: JsonObject) -> tuple[str | None, float | int]:
+    for tier in REWARD_TIERS:
+        value = tier_gradients.get(tier, 0)
+        if abs(float(value)) > 1e-12:
+            return tier, value
+    return None, 0
+
+
+def mean_policy_return_tuple(return_tuples: Sequence[Sequence[Any]]) -> list[float | int]:
+    if not return_tuples:
+        return [0 for _tier in REWARD_TIERS]
+    columns = list(zip(*return_tuples))
```

Ruff (0.15.12)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)